modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts

```diff
@@ -1402,3 +1402,35 @@ test(
 		await expect(accountSettingsPage.userPersonalMenuButton).toBeVisible();
 	}
 );
+
+test(
+	'Liferay allows more than 300kb profile picture into the user profile fix',
```

Comment:
Change this to something like
`Max fileSize limit is used uploading the user profile picture`